ts/src/program/index.ts

```diff
@@ -16,6 +16,11 @@ import { utf8 } from "../utils/bytes";
 import { EventManager } from "./event";
 import { Address, translateAddress } from "./common";
 
+export * from "./common";
```

Comment:
Do you need everything from these modules? I'd prefer to only export what's needed so that people don't rely on these internal APIs that may break/change in the future.